### PR TITLE
Edit homepage

### PR DESCRIPTION
**Pull request by bobturneruk**
Following yesterday's decision to make https://scottishcovidresponse.github.io/ our main web page, I propose some changes.

What I've done:
- Move website editing instructions to a separate area.
- Add key information from the University of Glasgow site.
- List our main achievements (for the benefit of those outside the organisation interested in what we've done).

Areas for work:
- Needs some stuff on visualisation work

### CODE REVIEW

**Comment by soniamitchell** [comment]
Looks reasonable to me.. though the original intention was to use this as an internal documentation hub, collating information about data, the registry, workflows, models, and so on, to make it easier to find information. With the UoG page being more public facing. Hence why the landing page contained editing instructions and the name "SCRC docs". If this is instead to be a public resource, I have no strong feelings either way. If RR is happy then it's fine by me.

**Comment by bobturneruk** [comment]
Please don't let my proposed changes make this into something less useful for it's intended purpose!

**Comment by soniamitchell** [comment]
I don't think it's less useful. My only thought is if it's internal only, then we already know who we are ;) If it's to be more of a public resource, then at the very least we'll need to be a bit more descriptive and provide context in our technical documentation. Again, I'm happy either way.

**Comment by richardreeve** [comment]
I agree that the original purpose was for this to be internal, but if we think of it more as a "technical" landing page vs our "public facing" landing page at UoG, then this information is all useful. We hope to have more people coming here for technical stuff as the data pipeline develops, and they may want to know who we are! @yabdalla1 is going to help sort out the potential conflict between the two sites.

**Comment by bobturneruk** [comment]
Sounds good. I won't merge without an approving review (I'd suggest protecting the `master` branch with this as a rule - I could do this). @yabdalla1 - are you able to edit this site, if you want to? Happy to help, if that's helpful.

File: content/docs/edit-site/edit-site.md
Comment (richardreeve) [inline content/docs/edit-site/edit-site.md]: Presumably need to say that you may not have edit permission on the site, and to email Sonia if you want to gain access.

File: content/_index.md
Comment (richardreeve) [inline content/_index.md]: I'm not sure there's any reason to single us out any longer, particularly on this site. I think we can either just say something like "formed by dozens of individuals from over 30 academic and commercial organisations" and link to the university at https://www.gla.ac.uk/research/az/scrc/ourpeople/#members , or (if you're really keen) list all of the organisations, starting with those three. I'd definitely go for the former though!

File: content/_index.md
Comment (richardreeve) [inline content/_index.md]: I don't think that's a good characterisation of what the APIs allow.
```suggestion
- Data APIs in five languages (python, Julia, R, Java and C++) that simplify provenance recording, allowing input data to be verified as it is used, and model outputs to be traced back to the model code and input data that were used to produce them.
```

**Review by richardreeve — CHANGES_REQUESTED** [review]
I think this is a good idea, but needs a couple of edits. I'm sure there's lots more to add of course, but this is just to correct the initial comments.

**Comment by richardreeve** [comment]
I'm not sure Yasmin has used github before, so may need to get up to speed - I'm certain she doesn't have write permission yet anyway. Good idea protecting master in the meanwhile (or whatever we want to change its name to given Sonia's earlier observation about branch names)...

**Comment by soniamitchell** [comment]
While we're making edits, it would be great to get those data pages populated. Particularly information regarding what data inputs the models take, perhaps linking to metadata from each dataset, which in turn might detail how data might be interpreted and how measurement might have changed throughout the course of the pandemic. Likely this would require contribution from data specialists and perhaps coordination with the data registry and data storage, to avoid replication of effort (assuming changes are being made on those platforms).

**Comment by richardreeve** [comment]
@soniamitchell Do you want to come to the modelling meeting tomorrow at 11 to discuss this?

**Comment by soniamitchell** [comment]
@richardreeve I'm happy to come along if you think I can be useful. Ruth might be a better shout though, as I believe she was working on collating the model data inputs during the RAMP period. 

I think the latter part of my comment might be a separate issue and might be more directly relevant to our current goals. That is, how dataset metadata might be presented and how we might integrate or distribute information from the data registry, data storage (are there ways to visualise this?), and the docs website.

**Comment by richardreeve** [comment]
I think it would be useful just to cover the first point (what data they are using) - it would be good to get the modellers involved in getting the site filled. Anyway, this isn't really relevant to this PR!

**Comment by soniamitchell** [comment]
Heh.. true! I've enabled the Discussions tab. Future chat can go there..

**Comment by bobturneruk** [comment]
@soniamitchell - Can I suggest using Zulip so we limit the places for discussion? I'm going to get confused otherwise!

File: content/_index.md
Comment (bobturneruk) [reply content/_index.md]: Thanks for this @richardreeve - my understanding of this is still evolving as I was previously mostly focussed on how the api interacted with a single model.

File: content/_index.md
Comment (bobturneruk) [reply content/_index.md]: Thanks - gone for the former. It's be great, I think, to get a list of people currently involved on this site so we can see who has what skills / who's doing what. Maybe I should raise an issue against this repo?

File: content/docs/edit-site/edit-site.md
Comment (bobturneruk) [reply content/docs/edit-site/edit-site.md]: I've added a hopefully adequate bit of text. I'll try and figure out how to give all organisation member write access so @soniamitchell Sonia isn't too distracted by requests.

**Comment by bobturneruk** [comment]
Hopefully this is now OK for now @richardreeve. Thanks for the feedback!

File: content/docs/edit-site/edit-site.md
Comment (soniamitchell) [reply content/docs/edit-site/edit-site.md]: Do we want everyone to have write access? If so, @richardreeve likely has a group set up for that..

.. or do we want to ask them to make a PR like what you’re doing here?

**Comment by soniamitchell** [comment]
> @soniamitchell - Can I suggest using Zulip so we limit the places for discussion? I'm going to get confused otherwise!

Fair enough

File: content/docs/edit-site/edit-site.md
Comment (bobturneruk) [reply content/docs/edit-site/edit-site.md]: I think they'd need write access to make a branch to get to the PR stage (unless they'd forked) then we'd want an approving review and we could potentially restrict who was authorised to approve changes. (I do get a bit lost with all the permissions on GitHub sometimes, they're quite complex but often don't let me do exactly what I want!)

**Review by soniamitchell — COMMENTED** [review]
Just built the site and it looks good. I have a couple of comments. Sorry I didn't catch them earlier..

Just an FYI... try to remember to update all the page weightings, you've missed `content/docs/research/_index.md`, which should be 4. Not a big deal now, but might cause an issue in the future.

Also, for some reason the menu didn't say "Editing Instructions" for me (like you've asked in `content/menu/index.md`) until I fiddled around for a while... I don't know if the menu was cached somehow or it's related to the lack of a title field in the header of `content/docs/edit-site/edit-site.md`. Admittedly, I'm a bit of Hugo novice.

And finally, it might be easier to add the menu items in the correct order, to avoid confusion, since I almost missed it!

File: content/docs/edit-site/edit-site.md
Comment (soniamitchell) [reply content/docs/edit-site/edit-site.md]: My only concern is breaking the site, so whatever you think is best (I'm confident you have more experience than me). Perhaps a dev branch that everyone can write to? Though part of me thinks most people might be more comfortable emailing Rmarkdown scripts / Word documents, I agree it would be easier if people could just fend for themselves.

**Comment by bobturneruk** [comment]
Thanks @soniamitchell - I am doubtless more novice than you here - I'm just applying general stuff I've learned from working with jekyll (another static site builder) and a little bit of hugo for other projects. Please let me know if it looks like if fixed my mistakes!

File: content/docs/edit-site/edit-site.md
Comment (bobturneruk) [reply content/docs/edit-site/edit-site.md]: I wouldn't be too confident of that! I suggest that the model is people make a branch, make changes, PR - this is reviewed by someone who can build the site locally (to make sure there are no breaking changes) and by someone who can offer a second opinion on the content. 

I think doing things by email would be great for people not comfortable with the above - we could make the branch and PR on their behalf.

Ultimately, if we get a lot of traffic, we might want to set up a CI step which does a test build of the site during the PR review process.

**Comment by soniamitchell** [comment]
It looks ok to me! Though I guess we won't know for sure until it's built. One other tiny thing.. on the landing page there's an extra space here `( RAPID ASSISTANCE`.

Perhaps we can learn together! 👍

**Comment by bobturneruk** [comment]
I hope so! I think the extra space is down to how the link is rendered, so I'll maybe try and fix later.

File: content/docs/edit-site/edit-site.md
Comment (soniamitchell) [reply content/docs/edit-site/edit-site.md]: Sounds like a good plan! Though from my experience last year, I think you're being a bit optimistic about the traffic :p

Comment (soniamitchell) [reply content/docs/edit-site/edit-site.md]: Alright, I've set most of that up, giving me, yourself, and Ryan write access to master. Reviews set up for everyone else. Just need @richardreeve to give the "everyone" team write access generally.

**Pull request merged**
